lib/scenario_runner.py

```diff
@@ -43,6 +44,12 @@
 from lib import container_compatibility
 from lib.container_compatibility import CompatibilityStatus
 
+class LogType(Enum):
```

Comment:
I introduced an enum type here. Probably more the Java than the Python way 😉
Let me know, if I should remove it.

Reply:
fine with that. we have that actually in multiple occurences and I personally find code more readable with that